Packages/com.unity.inputsystem/InputSystem/Controls/InputControlPath.cs

```diff
@@ -97,6 +97,19 @@ public static class InputControlPath
 
         public const char Separator = '/';
 
+        // We consider / a reserved character in control names. So, when this character does creep
```

Comment:
What if the original string contains space?

Reply:
Maybe its less likely that this string UTF-8(?) from lower character table is not present or are we sure there is no "ids" containing space?

Reply:
Spaces are fine. They have no special meaning in control paths.